dashboard_description_plugin/package.json

```diff
@@ -1,123 +1,130 @@
 {
     "name": "dashboard-description-plugin",
-    "version": "1.1.0",
+    "version": "1.1.1",
     "description": "A dashboard plugin for adding markdown description sections to a dashboard",
     "author": "Open-sourced by GoodData - feel free to change",
     "private": false,
     "license": "SEE LICENSE IN ../LICENSE",
-    "main": "dist/index.js",
-    "typings": "dist/index.d.ts",
+    "type": "module",
+    "exports": "./esm/index.js",
+    "types": "esm/index.d.ts",
     "sideEffects": [
         "*.css",
         "*.svg"
     ],
     "files": [
-        "dist/**/*.js",
-        "dist/**/*.json",
-        "dist/**/*.d.ts",
-        "dist/**/*.map",
-        "dist/**/*.svg"
+        "esm/**/*.js",
+        "esm/**/*.json",
+        "esm/**/*.d.ts",
+        "esm/**/*.map",
+        "esm/**/*.svg"
     ],
     "scripts": {
-        "clean": "rm -rf ci dist esm coverage *.log styles/css && jest --clearCache",
+        "clean": "rm -rf ci dist esm coverage *.log styles/css",
         "build": "webpack --mode production --config-name harness",
+        "prebuild-plugin": "yarn clean",
         "build-plugin": "webpack --mode production --config-name dashboardPlugin",
         "build-plugin-debug": "webpack --mode production --config-name dashboardPlugin --analyze",
         "gdc-plugins": "gdc-plugins",
         "add-plugin": "gdc-plugins dashboard-plugin add",
         "link-plugin": "gdc-plugins dashboard-plugin link",
         "unlink-plugin": "gdc-plugins dashboard-plugin unlink",
+        "update-plugin-params": "gdc-plugins dashboard-plugin update-params",
+        "remove-plugin-params": "gdc-plugins dashboard-plugin remove-params",
         "start": "webpack-cli serve --config-name harness",
-        "test": "jest --watch --passWithNoTests",
-        "test-once": "jest --maxWorkers=${JEST_MAX_WORKERS:-'45%'} --passWithNoTests",
-        "eslint": "eslint -c .eslintrc.js --ext ts,tsx src/",
+        "test": "vitest --watch --passWithNoTests",
+        "test-once": "vitest --passWithNoTests",
+        "eslint": "eslint -c .eslintrc.cjs src/",
+        "eslint-fix": "eslint -c .eslintrc.cjs --fix src/",
         "prettier-check": "prettier --check \"{src,test}/**/*.{ts,tsx,json,scss,md,yaml,html}\"",
         "prettier-write": "prettier --write \"{src,test}/**/*.{ts,tsx,json,scss,md,yaml,html}\"",
         "refresh-md": "node ./scripts/refresh-md.js"
     },
     "dependencies": {
-        "@gooddata/numberjs": "^4.0.3",
-        "@gooddata/sdk-ui-dashboard": "^8.10.0",
+        "@gooddata/number-formatter": "^1.0.2",
+        "@gooddata/sdk-ui-dashboard": "9.3.0",
```

Comment:
@dremileta can you update the SDK in the internal one too, also please test the PDF export after the update please as it has broken once the description sections layout in the PDF after dashboard layout changes.